webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -121,40 +135,63 @@ export const ErrataTab = () => {
         >
           <Thead>
             <Tr>
+              <Th key="expand-carat" />
               {columnHeaders.map(col =>
                 <Th key={col}>{col}</Th>)}
               <Th />
             </Tr>
           </Thead>
-          <Tbody>
-            {results?.map((erratum) => {
+          <>
+            {results?.map((erratum, rowIndex) => {
```

Comment:
{results?.map(({
                  id,
                  errata_id: errataId,
                  created_at: createdAt,
                  updated: publishedAt,
                  title,
                }, rowIndex) => <Tbody isExpanded={erratumIsExpanded(id)} key={`${id}_${createdAt}`}>...

Reply:
ditto regarding the above, lets compact this up a bit.

Reply:
Destructuring is only possible when the key and value are the same, no?  Not sure I'm getting what you're requesting

Reply:
You can do the above, I'm simply suggesting compacting this a bit.

Reply:
another correction, I see what you're doing below, ignore all the things!!